install/infra/terraform/eks/dns.tf

```diff
@@ -0,0 +1,114 @@
+resource "aws_route53_zone" "gitpod" {
+  force_destroy = true
+  name = var.domain_name
+
+  tags = {
+    Environment = "test"
```

Comment:
Ah no, as of now we do not depend on anything with these. I was thinking maybe it will be useful to cleanup orphan resources. But nothing is implemented.